pyproject.toml

```diff
@@ -0,0 +1,8 @@
+[build-system]
+requires = [
+    "setuptools",
```

Comment:
I'd lower-bound setuptools to avoid some issues we had in past. Although, I can't remember what those were, but I remember it was something. :smile:  Environment markers, right?

Reply:
Set it to `>=42.0.0` for now. A bit arbitrary but that's the version that started supporting pep518 so seemed natural :)

Reply:
I've found the lower-bound I had in mind -- looks like [it's 31.0 due to namespace packages support](https://github.com/dwavesystems/dwave-cloud-client/pull/43).

OTOH, I'm not sure where is 42.0.0 coming from? Looking at https://github.com/pypa/setuptools/issues/651, seems like PEP-518 support was not needed/not implemented in setuptools.

Reply:
Then again, I see from https://github.com/pypa/setuptools/commit/89155abb4222cf5a9dc81120e5c71e26b5af68f9 the full PEP-420 support (native namespace packages) are only supported since 40.1.0.

Reply:
I think the number came from [their changlog](https://setuptools.readthedocs.io/en/latest/history.html#v42-0-0).